src/stores/messages/MessagesStore.ts

```diff
@@ -128,6 +135,88 @@ export default class MessagesStore {
 		return [timestampTo - 15 * 1000, timestampTo + 15 * 1000];
 	}
 
+	public includes(message: EventMessage) {
```

Comment:
this is very confusing when you use names of built in methods.
`messagesStore.includes(message)` looks likes messagesStore is an array